test/e2e/scheduling/taints.go

```diff
@@ -19,20 +19,18 @@ package scheduling
 import (
 	"time"
 
+	. "github.com/onsi/ginkgo"
+	_ "github.com/stretchr/testify/assert"
```

Comment:
Keep these import separate

Reply:
+1. Please add a blank line after these imports and we are good to go.